packages/terra-form-select/tests/wdio/select-field-spec.js

```diff
@@ -0,0 +1,47 @@
+/* global before, browser, Terra */
+const viewports = Terra.viewports('tiny');
+
+describe('Select Field', () => {
+  describe('should select an option by click', () => {
```

Comment:
Since are only testing one viewport you could simplify this to be be

```
describe('Select Field', () => {
    before(() => browser.setViewportSize(Terra.viewports('tiny')[0]));
     describe('should select an option by click', () => {
         ... everything here can drop { viewports }
     });
});
```

Reply:
I think I'll keep this as it is, I like how it reads and keeps a consistent spot to update the viewports if we want to test this at different sizes